tests/test_pull.py

```diff
@@ -170,13 +167,13 @@ def test_pull_file(env):
     check_output(['git', 'big', 'pull', 'foo'])
 
     assert isfile(join(clone.repo_dir, 'foo'))
-    assert not isfile(join(clone.repo_dir, 'bar'))
```

Comment:
When we pull a single file here, we really want to be sure we didn't accidently pull the other file, which is why we check for `not isfile`. We shouldn't allow a link to be formed in this case.